source/sass/buyers-guide/fragments/_draft_product.scss

```diff
@@ -0,0 +1,14 @@
+// product draft rules
+&.draft::before {
+  content: 'This is a draft.';
+  color: #ff000054;
```

Comment:
Just a note that generally speaking we should always use 100% opaque colours (unless opacity is requested by design).